news/2020-06-19-Bootable-jar-Wildfly-20.adoc

```diff
@@ -0,0 +1,107 @@
+= A bootable JAR for WildFly 20
+jfdenise
+2020-06-19
+:awestruct-tags: [wildfly, galleon, bootable, jar, bootable-jar]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+
+== Introduction
+
+Now that WildFly supports link:https://projects.eclipse.org/projects/technology.microprofile[Eclipse Microprofile] this seems the appropriate time to introduce 
+a way to package WildFly application as bootable JAR (bootable JAR packaging is well suited for microservices to be run on cloud and bare metal).
+
+As we know (bootable or fat JAR are very common), a bootable JAR allows us to package both the container and the application in a single JAR file and execute it with a simple 
+command such as ```java -jar myapp-wildfly.jar```.
+
+Packaging the server and your app in a JAR makes sense when you can control the set of server features you want to see present. 
+Packaging a complete WildFly server (250+MB) for a simple microservice would defeat the purpose of the bootable JAR. link:https://docs.wildfly.org/galleon/[Galleon] technology 
+server trimming capabilities has made bootable JAR for WildFly a valid solution.
```

Comment:
s/made bootable/made a bootable/g

Reply:
I'd suggest to s/`capabilities has`/`capabilities have`, IIUC.